docs/contribute/review-issues.md

```diff
@@ -0,0 +1,66 @@
+---
+---
+
+{% capture overview %}
+
+This page explains how you should review and prioritize documentation issues made for the [kubernetes/kubernetes.github.io](https://github.com/kubernetes/kubernetes.github.io){: target="_blank"} repository. The purpose is to provide a way to organize issues and make it easier to contribute to Kubernetes documentation. The following should be used as the standard way of prioritizing, labeling, and interacting with issues.
```

Comment:
I think GH's raw markdown processor was a little wonky and italicized about half the document. If it looks fine in Netlify, LGTM.